packages/terra-base/src/BaseStyles.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react';
+import './Base.scss';
+
+const propTypes = {
+  /**
+   * The component(s) that will be wrapped by `<Base />`.
+   */
+  children: PropTypes.node.isRequired,
+};
+
+class BaseStyles extends React.Component {
+
+  componentDidMount() {
+    // Checks to run when not in production
+    if (process.env.NODE_ENV !== 'production') {
+      // Check to ensure terra-Base class is set on html element
+      if (!new RegExp('(^|\\s)terra-Base(\\s|$)').test(document.documentElement.className)) {
```

Comment:
Given that we're assuming that this is the root element, I'm curious about some scenarios we're trying to support.  We were going to potentially (in the interim phases for Orion) have multiple React components spread throughout the page amidst other non React elements.  Would terra-base still need to be at the root level in this scenario?  Or would it make sense to be at the root of each component.  I'm trying to figure out how this might work in our temporary hybrid world.